client/Sheepshead/SharedSheep/Game/Game.cs

```diff
@@ -38,6 +38,8 @@ private void DealCards(List<IPlayer> players)
             while (missingTrump)
             {
                 IDeck Deck = new Piquet();
+                Deck.CardsFactory();
```

Comment:
I don't like having to add the cards to the deck outside of the deck. When I create a new deck it should already contain all the cards it needs. Shuffle could be here if we wanted, but it doesn't have to be.